app/models/miq_request.rb

```diff
@@ -5,6 +5,7 @@ class MiqRequest < ApplicationRecord
   belongs_to :destination,       :polymorphic => true
   belongs_to :requester,         :class_name  => "User"
   belongs_to :tenant
+  belongs_to :service_order
```

Comment:
No, I do not think that is needed either.  Could always be a data migration at a later time if we change our minds.